sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/yaml/YAMLToInternalMappers.java

```diff
@@ -358,6 +327,15 @@ private static PodSpec convertPod(
       builder.hostVolumes(hostVolumeSpecs);
     }
 
+    if (!rawPod.getExternalVolumes().isEmpty()) {
+      Collection<HostVolumeSpec> externalVolumeSpecs = new ArrayList<>();
+      externalVolumeSpecs.addAll(rawPod.getExternalVolumes().values().stream()
+              .map(v -> convertHostVolume(v))
+              .collect(Collectors.toList()));
+
+      builder.hostVolumes(externalVolumeSpecs);
```

Comment:
```suggestion
      builder.externalVolumes(externalVolumeSpecs);
```

I think you meant `builder.externalVolumes` which you defined in this PR here intead of `builder.hostVolumes`